sdk/eventgrid/azure-eventgrid/azure/eventgrid/_client.py

```diff
@@ -0,0 +1,78 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from copy import deepcopy
+from typing import Any
+
+from azure.core import PipelineClient
+from azure.core.credentials import AzureKeyCredential
+from azure.core.rest import HttpRequest, HttpResponse
+
+from ._configuration import EventGridClientConfiguration
+from ._operations import EventGridClientOperationsMixin
+from ._serialization import Deserializer, Serializer
+
+
+class EventGridClient(EventGridClientOperationsMixin):  # pylint: disable=client-accepts-api-version-keyword
+    """Azure Messaging EventGrid Client.
+
+    :param endpoint: The host name of the namespace, e.g.
+     namespaceName1.westus-1.eventgrid.azure.net. Required.
+    :type endpoint: str
+    :param credential: Credential needed for the client to connect to Azure. Required.
+    :type credential: ~azure.core.credentials.AzureKeyCredential
+    :keyword api_version: The API version to use for this operation. Default value is
+     "2023-06-01-preview". Note that overriding this default value may result in unsupported
+     behavior.
+    :paramtype api_version: str
+    """
+
+    def __init__(self, endpoint: str, credential: AzureKeyCredential, **kwargs: Any) -> None:
+        _endpoint = "{endpoint}"
+        self._config = EventGridClientConfiguration(endpoint=endpoint, credential=credential, **kwargs)
+        self._client: PipelineClient = PipelineClient(base_url=_endpoint, config=self._config, **kwargs)
+
+        self._serialize = Serializer()
+        self._deserialize = Deserializer()
+        self._serialize.client_side_validation = False
+
+    def send_request(self, request: HttpRequest, **kwargs: Any) -> HttpResponse:
```

Comment:
does this generated client not have a wrapper? it looks like you call methods that are not listed here in the samples?

Reply:
in _patch.py